components/cstudio-forms/controls/checkbox-group.js

```diff
@@ -212,7 +212,7 @@ YAHOO.extend(CStudioForms.Controls.CheckBoxGroup, CStudioForms.CStudioFormField,
                     groupEl.appendChild(rowEl);
                 }
                 _self.value = newValue;
-                _self.form.updateModel(_self.id, _self.getValue());
+                //_self.form.updateModel(_self.id, _self.getValue());
 
```

Comment:
Has this been fully tested.  It looks weird.  Why does this solve the problem?
Also, don't comment out code.  remove it.